scurl/cgurl.pyx

```diff
@@ -138,7 +135,8 @@ cdef object extra_attr(obj, prop, bytes url, Parsed parsed, decoded, params=Fals
             try:
                 port = int(port, 10)
             except ValueError:
-                message = f'Port could not be cast to integer value as {port!r}'
+                # change to format() to support pypy
+                message = 'Port could not be cast to integer value as {}'.format(repr(port))
```

Comment:
I originally got this piece of code from [here](https://github.com/python/cpython/blob/master/Lib/urllib/parse.py#L173) but f string is apparently not supported on pypy py2.7, so I changed it to use format() instead. Do you think that's okay @lopuhin ?

Reply:
Sure! ``f""`` syntax was added only in Python 3.6